ix-dev/community/lms/app.yaml

```diff
@@ -0,0 +1,32 @@
+app_version: 9.0.0
+capabilities: []
```

Comment:
Missing the capabilities here (chown, etc)

Reply:
done